app/views/affiliate_requests/email_link_status.html.erb

```diff
@@ -3,7 +3,7 @@
     <h1>Affiliates</h1>
   </header>
   <div>
-    <div class="placeholder">
+    <div class="grid gap-3 rounded border border-dashed border-border p-6 text-center justify-items-center bg-filled">
```

Comment:
Please add a comment to use the Placeholder component when we migrate this to React